README.md

```diff
@@ -239,24 +239,290 @@ One can also do it manually:
 
       $ dotnet new -u Equinox.Templates
 
+<a name="guidance"></a>
 # PATTERNS / GUIDANCE
 
-## Use Strongly typed ids
+<a name="tldr"></a>
+## TL;DR
 
-Wherever possible, the samples strongly type identifiers, particularly ones that might naturally be represented as primitives, i.e. `string` etc.
+1. DO have global strongly typed id types in `namespace Domain`
+2. DONT have global `module Types`. AVOID per Aggregate `module Types` or top level `type` definitions
+3. DO group stuff predictably per `module Aggregate`: `Stream, Events, Reactions, Fold, Decisions, Service, Factory`. And keep grouping within that.
+4. DONT `open <Aggregate>`, `open <Aggregate>.Events`, `open <Aggregate>.Fold`
```

Comment:
counterpoint

```fsharp
module Fold = 
    open Events
```

Reply:
also useful to `open Aggregate` in test files, but can do without

Reply:
I didnt call those out further down but I will. I also intend to link these tl;dr sentences to sections below

Reply:
Hopefully https://github.com/jet/dotnet-templates/pull/133/commits/c1a7ff54dbe288b5f3eeaa6c4b2fbe9e2ffc0e23 covers it